etl/pad/raw/skills/leader_skill_info.py

```diff
@@ -1496,7 +1496,7 @@ def __init__(self, ms: MonsterSkill):
         self.min_rcv = 1
         self.rcv_step = 0
 
-        max_atk = self.min_atk + self.atk_step * (5 - data[5])
+        max_atk = self.min_atk + self.atk_step * len(binary_con(data[5]))
```

Comment:
Okay, I have NO clue what `data[5]` is either, but this logic doesn't feel right.  I don't think this is a bitflag.  I believe the right logic here is to use `(5 - data[1])`, or more clearly, `(self.max_attr - self.min_attr)`.  I would also make another attribute called `self.unknown5` and set it to `data[5]`.  I really do have NO clue what it does.  It's also of note that making this change will have no effect.  The only monster that uses `data[5]` with a non-zero `data[4]` is Gorkaos, and 3 is the minimum number of attributes.

Reply:
Okay.  I think this should be `(self.max_attr - self.min_attr)`